### ISSUE TITLE
Back button skips a page when searching Google

### ISSUE THREAD

**Issue by wickedsp1d3r** [andryou/scriptsafe#1]
Hi,

I have noticed that this extension causes an issue with Google search: when I search something and click a link -> open the website -> click browser's back button -> Google search goes to the previous search that I possibly did before this search I am talking. It basically skips one search.

Everything is working normally when I have ScriptSafe turned off, so this problem has to be related to ScripSafe. Whilisting does not work.

Hope you are still developing ScriptSafe and could look for this issue. Thanks!

**Comment by andryou** [comment]
Thanks! Can you please let me know if this issue is still occurring in v1.0.6.19?

**Comment by ROpdebee** [comment]
I've never had any issues of the like before the update to 1.0.6.19 but since I've updated I've been getting some weird issues with the back button going back to pages from _other tabs_ that were recently closed (not even the last one either). Started occurring after the update to 1.0.6.19 but it might also be a Chrome  51 issue as I updated both Chrome and ScriptSafe at the same time.

It's only happened two times so far but that's twice in a 24h timeframe. Will keep an eye out for it again and report back if it happens to me again. At first I actually thought it was a Chrome bug until I saw this issue, although my mind would be blown if this is the same issue and it'd taken 2 years to surface for me and then occurred twice.
